lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1795,7 +1795,8 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
   }
 
   // Translation of instructions
-  switch (BV->getOpCode()) {
+  int OpCode = BV->getOpCode();
+  switch (OpCode) {
```

Comment:
Why do you need this change? If I understand correctly, `internal::OpAssumeTrueINTEL` should already have the same type as `OpBranch`, i.e. `Op`, so no explicit conversion to `int` should be needed to align types. Am I missing something?

Reply:
`internal::OpAssumeTrueINTEL` isn't member of `spv::Op`, instead it's a member of `spv::internal::Op`, hence there is a compiler warning.

Reply:
Double check:
/localdisk2/sidorovd/xmain/ics-ws/SYCLOS/llvm/llvm-spirv/lib/SPIRV/SPIRVReader.cpp:2412:3: warning: case value '5630' not in enumerated type 'spv::Op' [-Wswitch]
   case internal::OpAssumeTrueINTEL: {

Reply:
Double check:
lib/SPIRV/SPIRVReader.cpp:2412:3: warning: case value '5630' not in enumerated type 'spv::Op' [-Wswitch]
case internal::OpAssumeTrueINTEL: {

Reply:
Got, thanks